server/services/educationAndWorkPlanService.test.ts

```diff
@@ -21,49 +21,57 @@ import toEducationDto from '../data/mappers/educationMapper'
 import aValidEducationDto from '../testsupport/educationDtoTestDataBuilder'
 import aValidCreateEducationDto from '../testsupport/createEducationDtoTestDataBuilder'
 import aValidCreateEducationRequest from '../testsupport/createEducationRequestTestDataBuilder'
+import HmppsAuthClient from '../data/hmppsAuthClient'
 
 jest.mock('../data/mappers/educationMapper')
 jest.mock('../data/educationAndWorkPlanClient')
 jest.mock('./prisonService')
+jest.mock('../data/hmppsAuthClient')
 
 describe('educationAndWorkPlanService', () => {
   const educationAndWorkPlanClient =
     new EducationAndWorkPlanClient() as unknown as jest.Mocked<EducationAndWorkPlanClient>
   const prisonService = new PrisonService(null, null, null) as unknown as jest.Mocked<PrisonService>
-  const educationAndWorkPlanService = new EducationAndWorkPlanService(educationAndWorkPlanClient, prisonService)
 
   const mockedEducationMapper = toEducationDto as jest.MockedFunction<typeof toEducationDto>
+  const systemToken = 'a-system-token'
 
   beforeEach(() => {
     jest.resetAllMocks()
+    hmppsAuthClient.getSystemClientToken.mockResolvedValue(systemToken)
   })
 
+  const hmppsAuthClient = new HmppsAuthClient(null) as jest.Mocked<HmppsAuthClient>
+  const educationAndWorkPlanService = new EducationAndWorkPlanService(
+    educationAndWorkPlanClient,
+    prisonService,
+    hmppsAuthClient,
+  )
+
   describe('createGoals', () => {
     it('should create Goals', async () => {
       // Given
       const prisonNumber = 'A1234BC'
-      const userToken = 'a-user-token'
       const createGoalDto = aValidCreateGoalDtoWithOneStep()
       const createGoalsRequest = aValidCreateGoalsRequestWithOneGoal()
 
       // When
-      await educationAndWorkPlanService.createGoals(prisonNumber, [createGoalDto], userToken)
+      await educationAndWorkPlanService.createGoals(prisonNumber, [createGoalDto], systemToken)
```

Comment:
No, this isn't quite right.
It's only the `getEducation` method of `educationAndWorkPlanService` that you're changing to use the system token.
All other service methods in `educationAndWorkPlanService` are still using the user token, so you dont want to make these changes.

Reply:
Fixed.